src/dbnode/storage/cleanup.go

```diff
@@ -124,7 +124,7 @@ func newCleanupManager(
 	}
 }
 
-func (m *cleanupManager) Cleanup(t time.Time, isBootstrapped bool) error {
+func (m *cleanupManager) WarmFlushCleanup(t time.Time, isBootstrapped bool) error {
```

Comment:
Hm, this method is more like "CleanupIndexAndSnapshotFiles" right? It doesn't have warm data data files being cleaned up?

I understand that it's sync'd with the WarmFlush lifecycle, but maybe better to describe what it does than what the lifecycle is?